.github/workflows/push.yml

```diff
@@ -46,11 +46,11 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     - name: Create Release
       if: startsWith(github.ref, 'refs/tags/v')
-      uses: actions/create-release@v1
+      uses: ncipollo/release-action@v1
       with:
-        tag_name: ${{ github.ref }}
-        release_name: ${{ github.ref }}
+        name: ${{ env.VERSION }}
         body: ${{steps.build_changelog.outputs.changelog}}
         prerelease: "${{ contains(github.ref, '-rc') }}"
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        # Ensure target branch for release is "master"
+        commit: master
```

Comment:
Are you sure this is what you wanted?
The readme says
> commit: An optional commit reference. This will be used to create the tag if it does not exist.

Reply:
I checked the implementation to ensure the value is used to specify the target for the release as discussed in the linked community post, and testing on my fork shows that it works as intended: https://github.com/simu/commodore/releases/tag/v0.7.1